### DISCUSSION TITLE
Mojo Demo Lectures

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/meta#115] [Projects]
@QuantEcon/high-performance The following repository 

https://github.com/QuantEcon/project.lecture-mojo

contains the demo site for the `QuantEcon` mojo examples and tutorials. 

It would be great to set the `gh-pages` site up and running

@kp992 or @HumphreyYang would you have time to:

- [ ] understand the current execution issues locally https://github.com/QuantEcon/project.lecture-mojo/issues/17 and apply fixes
- [ ] understand the execution issues on `gh-action` builds (the build artefacts are now available for each run)

**Comment by mmcky** [comment]
One the `local` issues are fixed then I think we can focus on the cloud side of things. It looks like there is an issue with `nbclient` connections at the moment but I won't debug further until the local side is verified.

**Comment by kp992** [comment]
Thanks for reporting @mmcky. I'll have a look at it by this weekend.

**Comment by kp992** [comment]
Hi @mmcky ,

I tried fixing the local build issue. The nbclient issue on CI still exists due to some reason. Locally, it works fine for me:
```bash
% jupytext --to ipynb applications/shortest_paths.md
[jupytext] Reading applications/shortest_paths.md in format md
[jupytext] Writing applications/shortest_paths.ipynb (destination file replaced [use --update to preserve cell outputs and ids])

% jupyter nbconvert --execute --to html applications/shortest_paths.ipynb
[NbConvertApp] Converting notebook applications/shortest_paths.ipynb to html
[6705:98793:20231230,164628.234605:ERROR directory_reader_posix.cc:42] opendir /Users/kpl/.modular/crashdb/attachments/76396b1a-1e4d-4566-856c-1c58c0bdcd98: No such file or directory (2)
[NbConvertApp] Writing 297757 bytes to applications/shortest_paths.html
```

**Reply by kp992** [reply]
I tried debugging on the CI but seems some issue with kernel connection?
